rules-tests/Php80/Rector/Switch_/ChangeSwitchToMatchRector/Fixture/copy_switch_comment.php.inc

```diff
@@ -1,40 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php80\Rector\Switch_\ChangeSwitchToMatchRector\Fixture;
-
-class CopySwitchComment
-{
-    public function run()
-    {
-        // some comment here
```

Comment:
this test might be re-added later with a proper example. I played it safe for now and removed it because of the switch-on-mixed assumption